### ISSUE TITLE
Weird condition in Sdk.zig

### ISSUE THREAD

**Issue by leecannon** [MasterQ32/SDL.zig#63]
The below if conditions are clearly wrong, shouldn't `detected_target` be involved?

https://github.com/MasterQ32/SDL.zig/blob/9bef3552c4a031193a79884107466ea6ce8a5f4f/Sdk.zig#L235-L236

**Comment by MasterQ32** [comment]
Yep, that's true. I guess i have to recreate that part of the logic, it's pretty complex sadly
